charts/prometheus-statsd-exporter/templates/deployment.yaml

```
@@ -16,6 +16,7 @@ spec:
     metadata:
       {{- with .Values.podAnnotations }}
       annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
         {{- toYaml . | nindent 8 }}
       {{- end }}
```

Thank you, @gavin-jeong. In this way, `checksum/config` has been made dependent on `podAnnotations` which is probably not desired. What if we took the annotation out of the `with` block:

```suggestion
      annotations:
        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
        {{- with .Values.podAnnotations }}
        {{- toYaml . | nindent 8 }}
        {{- end }}
```

---

Looks better : )